examples/injection/module/rust/Cargo.toml

```diff
@@ -12,17 +12,12 @@ crate-type = ["cdylib"]
 anyhow = "*"
 log = "*"
 oak_abi = "=0.1.0"
+oak_io = "=0.1.0"
 oak_services = "=0.1.0"
 prost = "*"
 
 [dependencies.oak]
```

Comment:
Could this now be changed to `oak = { version = "=0.1.0", features = ["linear-handles"] }` to be more consistent throughout.

Reply:
Yes, good point! Done.